tests/js/panelsTests.js

```diff
@@ -242,6 +292,24 @@ https://github.com/gpii/universal/LICENSE.txt
         jqUnit.assertEquals("The model value for the language is set correctly", expected, that.model.lang);
     };
 
+    gpii.tests.langTester.findLangRow = function (that, lang) {
+        return that.locate("langRow").has("[lang='" + lang + "']");
+    };
+
+    gpii.tests.langTester.focus = function (that, lang) {
+        var button = gpii.tests.langTester.findLangRow(that, lang);
+        fluid.focus(button);
+    };
+
+    gpii.tests.langTester.keydown = function (that, lang, keyCode) {
+        var button = gpii.tests.langTester.findLangRow(that, lang);
+        gpii.tests.utils.simulateKeydown(button, keyCode);
+    };
+
+    gpii.tests.langTester.verifySelectedLangModel = function (that, lang) {
+        jqUnit.assertEquals("The model value for \"selectedLang\" has been set correctly", lang, that.model.selectedLang);
```

Comment:
Maybe add the expected lang in the test message, to make it easier to distinguish between the tests.